index.html

```diff
@@ -12904,10 +12904,10 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 		<div class="property" id="aria-setsize">
 			<pdef>aria-setsize</pdef>
 			<div class="property-description">
-				<p><a>Defines</a> the number of items in the current set of listitems or treeitems. Not required if all elements in the set are present in the <abbr title="Document Object Model">DOM</abbr>. See related <pref>aria-posinset</pref>.</p>
-				<p>This [=ARIA/property=] is marked on the members of a set, not the container element that collects the members of the set. To orient the user by saying an element is "item X out of Y," the <a>assistive technologies</a> would use X equal to the <pref>aria-posinset</pref> <a>attribute</a> and Y equal to the <code>aria-setsize</code> attribute.</p>
+				<p><a>Defines</a> the number of items in the current set of items, such as listitems, treeitems, options, comments, etc. Not required if all elements in the set are present in the <abbr title="Document Object Model">DOM</abbr>. See related <pref>aria-posinset</pref>.</p>
+				<p>This [=ARIA/property=] is marked on the members of a set, not the container element that collects the members of the set. To orient the user by saying an element is "item X out of Y" or similar, the <a>assistive technologies</a> would use X equal to the <pref>aria-posinset</pref> <a>attribute</a> and Y equal to the <code>aria-setsize</code> attribute.</p>
 				<p>If all items up to the current item in a set are present in the document structure, it is not necessary to set this <a>attribute</a>, as the <a>user agent</a> can automatically calculate the position for these items. However, if all previous items in the set are not present in the document structure at a given moment, the author MUST set this <a>attribute</a>to provide an explicit indication of an element's position.</p>
-				<p>When specifying <code>aria-setsize</code>, authors MUST set the value to an integer equal to the number of items in the set. If the total number of items is unknown, authors SHOULD set the value of <code>aria-setsize</code> to <code>-1</code>.</p>
+				<p>When specifying <code>aria-setsize</code>, authors MUST set the value to an integer equal to the number of items in the set. If the total number of items is unknown (indeterminate), and the DOM is currently incomplete such that user agents would not be able to compute the correct number of items in a set, authors SHOULD set the value of <code>aria-setsize</code> to a negative number (e.g., <code>-1</code>) or <code>0</code>.</p>
```

Comment:
We probably should just keep recommending -1 because some implementations may not update soon, and -1 was previously the way.

That doesn't mean the AAM wouldn't handle 0, but there's no reason to give authors extra options here.